test/fixtures/create-order-response-parsed.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
When we update the order endpoint to be inline with the changes in `develop` on the platform, then I think we should update these fixtures to use a more complete data set.

Reply:
👍